core/cert.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
+MIIDADCCAeigAwIBAgIQBn5uTveKrtoktTU4oxRXjDANBgkqhkiG9w0BAQsFADAS
```

Comment:
do we want this in here or just for your wip?

Reply:
done